packages/fx-core/resource/package.nls.json

```diff
@@ -1017,6 +1017,14 @@
   "configManager.error.driverNotFound": "Driver %s not found for step %s",
   "configManager.error.yamlParsing": "Error parsing yaml file at path %s. Reason: %s",
   "configManager.error.invalidLifecycle": "The schema of lifecycle %s is invalid",
+  "botRegistration.ProgressBar.creatingBotAadApp": "Creating bot aad app.",
+  "botRegistration.log.startCreateBotAadApp": "Creating bot aad app.",
+  "botRegistration.log.successCreateBotAadApp": "Bot aad app created successfully.",
+  "botRegistration.log.skipCreateBotAadApp": "Bot aad app creation skipped.",
+  "botRegistration.log.failCreateBotAadApp": "Failed to create bot aad app. Error message: %s.",
+  "driver.botAadApp.log.startExecuteDriver": "Executing driver %s",
```

Comment:
I think we are changing the name "driver" to "action", can you confirm @blackchoey ?

Reply:
Yes, I need to drive this change...

Reply:
@IvanJobs can you use `action` instead of `driver` in your PR first? thanks